frontend/jupyter/src/app/pages/index/index-default/index-default.component.html

```diff
@@ -1,76 +1,81 @@
 <div class="lib-content-wrapper">
-  <lib-title-actions-toolbar title="Notebooks" [buttons]="buttons" i18n-title>
-    <lib-namespace-select
-      *ngIf="(ns.dashboardConnected$ | async) === dashboardDisconnectedState"
-      namespacesUrl="/api/namespaces"
-    ></lib-namespace-select>
-  </lib-title-actions-toolbar>
-
-  <!--scrollable page content-->
-  <div class="page-padding lib-flex-grow">
-    <lib-table
-      [config]="config"
-      [data]="processedData"
-      [trackByFn]="notebookTrackByFn"
-      (actionsEmitter)="reactToAction($event)"
-    ></lib-table>
+  <div *ngIf="!notebookInfoLoaded">
```

Comment:
the spinner should probably appear below the namespace selector, and the namespace select should probably be outside of the new ng-container, as the spinner is just about when the notebooks are loading, not the namespace information (and this was probably the cause of the cypress tests initially failing).